### ISSUE TITLE
Regarding the Scale Normalization of Point Clouds in the "test_criterion" Loss Function of I2P Network During Training

### ISSUE THREAD

**Issue by zhangshuoneu** [PKU-VCL-3DV/SLAM3R#37]
Hello, author! This is a great and very interesting open-source project. While trying to understand the training code, I have some confusion about the test_criterion parameter configuration for I2P in scripts/train_i2p.sh.

The paper mentions that the I2P network performs scale normalization on both the predicted point cloud and the ground truth (GT) point cloud. However, in the I2P training configuration, --test_criterion "Jointnorm_Regr3D(L21, gt_scale=True)", setting gt_scale=True causes the GT point cloud in slam3r/losses.py to use its original scale. My question is: why is the GT point cloud not normalized during testing?

![Image](https://github.com/user-attachments/assets/7f029118-121a-413e-bede-f7fcaf1d24a3)

**Comment by zhangshuoneu** [comment]
Sorry, my mistake! There is no problem!
